dspback/pydantic_schemas.py

```diff
@@ -67,7 +67,7 @@ class Submission(Document):
     identifier: str = None
     submitted: datetime = datetime.utcnow()
     url: HttpUrl = None
-    metadata_json: str = {}
+    metadata_json: str = "{}"
```

Comment:
Are we storing this as string b/c its some kind of for archival purposes? I guess we can set the default to be `None`.

Reply:
It is a string because that is what the metadata was originally stored in when we were using postgresql. It could be updated to not be a string but not sure if it is worth the effort right now.